apps/frontend/src/pages/auth/components/sign-up-form/style.module.css

```diff
@@ -1,33 +1,140 @@
-.signup-container .logo-container {
+.container {
+	position: relative;
+	display: grid;
+	grid-template-columns: 1fr 1fr;
+	gap: 30px;
+	height: 100dvh;
+	padding: 15px;
+	overflow: hidden;
+	background-color: var(--background);
+}
+
+.form-container {
+	z-index: var(--z-index-low);
 	display: flex;
-	gap: 8px;
 	align-items: center;
+	justify-content: center;
+	background-color: var(--white);
+	border-radius: 20px;
 }
 
-.circle {
-	width: 30px;
-	height: 30px;
-	border-radius: 100%;
+.form {
+	display: flex;
+	flex-direction: column;
+	gap: 20px;
+	width: 100%;
+	max-width: 350px;
+	padding-inline: 20px;
 }
 
-.circle-pink {
-	background: var(--pink-gradient);
+.form-header {
+	margin-bottom: 5px;
 }
 
-.title-form {
-	margin-top: 30px;
-	margin-bottom: 8px;
-	font-weight: 700;
+.form-header__logo-container {
+	display: flex;
+	gap: 7px;
+	align-items: center;
+	margin-bottom: 35px;
 }
 
-.signup-container .auth-info {
-	font-family: inherit;
-	font-weight: 600;
+.form-header__logo {
+	width: 33px;
+	height: 33px;
+	background: linear-gradient(
+		to right,
+		var(--free-time-start) 0%,
+		var(--free-time-end) 100%
+	);
+	border-radius: 50%;
 }
 
-.input-groups {
+.form-header__logo-text {
+	font-size: 24px;
+	font-weight: bold;
+	text-transform: uppercase;
+}
+
+.form-header__text {
+	margin-block: 10px;
+	text-transform: uppercase;
+}
+
+.form-header__sub-text {
+	font-size: medium;
```

Comment:
I think you should use	font-size with px

Reply:
Agree, also I believe `medium` is the default one